src/fileseq/filesequence.py

```diff
@@ -312,6 +396,8 @@ def invertedFrameRange(self):
         """
         if not self._frameSet:
             return ''
+        if self._decimal_places:
+            return ''
```

Comment:
Yeah, fair enough. Technically it is possible to invert a frame range as long as all frames differ by a multiple of the same stride, but it would be complicated to implement the general case as first you have to calculate that base stride, and honestly who would need to do that? Will think about how difficult this would be, or add a comment about not supporting it.